common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -98,6 +98,8 @@ class WhiskConfig(requiredProperties: Map[String, String],
   val controllerSeedNodes = this(WhiskConfig.controllerSeedNodes)
   val controllerLocalBookkeeping = getAsBoolean(WhiskConfig.controllerLocalBookkeeping, false)
   val controllerHighAvailability = getAsBoolean(WhiskConfig.controllerHighAvailability, false)
+  val controllerProtocol = this(WhiskConfig.controllerProtocol)
+  val controllerSSLEnabled = if (this.controllerProtocol == "https") true else false
```

Comment:
Can we move this to pureconfig as well?

Reply:
I wanted to keep it in the Config, so we don't create two identical objects in Trigger and Controller or pass that property around.

Reply:
As we need to go the internal way for triggers (long term), I think it's more valuable to use pureconfig here instead of the old whiskConfig.